packages/nimble-components/src/table-column/number-text/models/byte-scale-formatter.ts

```diff
@@ -0,0 +1,70 @@
+import { IntlNumberFormatScaledUnit } from './intl-number-format-scaled-unit';
+import type { ScaledUnit } from './scaled-unit';
+import { UnitScaleFormatter } from './unit-scale-formatter';
+
+/**
+ * Formatter for numbers with byte units (1000-based)
+ */
+export class ByteScaleFormatter extends UnitScaleFormatter {
```

Comment:
There are some basic naming consistency issues around all these. Like this is arbitrarily leaving off `Unit` I'd vote for `ByteUnitScaleFormatter`. The thing that this is is not a "ScaleFormatter" (there isn't such a thing), it is a "UnitScaleFormatter"